src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -0,0 +1,146 @@
+package nextstep.subway.line.domain;
+
+import nextstep.subway.exception.NotValidDataException;
+import nextstep.subway.station.domain.Station;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static nextstep.subway.exception.type.ValidExceptionType.*;
+
+@Embeddable
+public class Sections {
+    @OneToMany(mappedBy = "line", cascade = CascadeType.ALL, orphanRemoval = true)
+    private final List<Section> sections = new ArrayList<>();
+
+    protected Sections() {
```

Comment:
```suggestion
    protected Sections() {}
```
이 부분만 개행이 되어 있는 것 같습니다. 아마 살짝 놓치신 것 같아요...! 
그리고 [컨벤션](https://naver.github.io/hackday-conventions-java/#permit-concise-empty-block)과 같이, 다른 코드 처럼 한줄에 작성해주셔도 좋을 것 같습니다..!